WebApp/src/main/java/ev/projects/webapp/runners/SeedRunner.java

```diff
@@ -0,0 +1,40 @@
+package ev.projects.webapp.runners;
+
+import ev.projects.models.User;
+import ev.projects.repositories.ICaseRepository;
+import ev.projects.services.IDocumentService;
+import ev.projects.services.IUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import static ev.projects.models.CaseFactory.createCase;
+import static ev.projects.models.DocumentFactory.createAttachmentOfDocument;
+import static ev.projects.models.DocumentFactory.createDocumentOfCase;
+
+@Component
+@Profile("!test")
+public class SeedRunner implements CommandLineRunner {
```

Comment:
What is the purpose of this class. I suppose it creates some kind infrastructure for tests?
Not sure what you agreed with Jurij on comments. But javadoc format comments would be very helpful to clear out the purpose of classes and public methods - not only in this class but all the project.

Reply:
This class is not used in tests (annotations `@Profile("!test")` in SeedRunner and `@ActiveProfiles("test")` in test classes prevent that). However it is used by the simulator. I though it would be tedious on app start to register a new user via Postman so I need data at the start. Since it's a simulator I think it's not a problem.
About javadoc comments, I don't remember him mentioning them, however I will document the code then.

Reply:
Should I only document interfaces or implementation classes also?

Reply:
I mean for example I have `    @Override
    public Optional<Case> getById(Long ID) {
        return caseRepository.findById(ID);
    }` in ICaseService implementation class CaseService. Let's assume another implementation is added which for some reason would also getById given the ID, but would have more calculations. Should these additional calculations be documented in impl class?

Reply:
> Should I only document interfaces or implementation classes also?

Usually only interface methods are commented, as these are the ones called. Implementation comments are optional - mostly for places with complex custom logic.

Reply:
The issue of documentation is too extensive for this branch. Have escalated it to #6.

Reply:
Whoops. It has a tail of this branch. :D
Will create a new PR when this is merged.

Reply:
> I mean for example I have ` @Override public Optional<Case> getById(Long ID) { return caseRepository.findById(ID); }` in ICaseService implementation class CaseService. Let's assume another implementation is added which would also getById given the ID, but for some reason would have more calculations. Should these additional calculations be documented in impl class?

Usually it is enough to add short description what the method does, what parameters are taken and what are returned. The internal implementation does not matter at this point. Some of more complex functionality can be commented inside these implementation classes separately if it is worth to comment. Obviously no need to comment getters and setters.

Reply:
> Whoops. It has a tail of this branch. :D Will create a new PR when this is merged.

Ok, no problem